resources/views/events/view.blade.php

```diff
@@ -1,290 +1,363 @@
+          <?php
+          // We need to expand the user objects to pass to the client.  In due course this will be replaced
+          // by an API call to get the event details.
+          $expanded_attended = [];
+          foreach ($attended as $att) {
+            $thisone = $att;
+            $thisone['volunteer'] = $att->volunteer;
+            $thisone['userSkills'] = $att->volunteer->userSkills;
+            $thisone['fullName'] = $att->getFullName();
+            $thisone['profilePath'] = $att->volunteer->getProfile($att->id)->path;
+            $expanded_attended[] = $thisone;
+          }
+          $expanded_invited = [];
+          foreach ($invited as $att) {
+            $thisone = $att;
+            $thisone['volunteer'] = $att->volunteer;
+            $thisone['userSkills'] = $att->volunteer->userSkills;
+            $thisone['fullName'] = $att->getFullName();
+            $thisone['profilePath'] = $att->volunteer->getProfile($att->id)->path;
+            $expanded_invited[] = $thisone;
+          }
+
+          $expanded_hosts = [];
+          foreach ($hosts as $host) {
+            $thisone = $host;
+            $thisone['volunteer'] = $host->volunteer;
+            $expanded_hosts[] = $thisone;
+          }
+
+          $attendance_edit = (FixometerHelper::hasRole(Auth::user(), 'Host') && FixometerHelper::userHasEditPartyPermission($formdata->id, Auth::user()->id)) || FixometerHelper::hasRole(Auth::user(), 'Administrator');
+          ?>
+
+          <div class="d-flex flex-wrap">
+            <div class="w-xs-100 w-md-50">
+              <div class="vue">
+                <EventDetails class="pr-md-3" :event-id="{{ $event->idevents }}" :event="{{ $event }}" :hosts="{{ json_encode($expanded_hosts) }}" :calendar-links="{{ json_encode($calendar_links) }}" />
+              </div>
+              <div class="vue">
+                <hr class="d-block d-md-none">
+                <EventDescription class="pr-md-3" :event-id="{{ $event->idevents }}" :event="{{ $event }}" />
+              </div>
+            </div>
+            <div class="w-xs-100 w-md-50 vue">
+              <EventAttendance class="pl-md-3" :event-id="{{ $event->idevents }}" :event="{{ $event }}" :attendance="{{ json_encode($expanded_attended) }}" :invitations="{{ json_encode($expanded_invited) }}" :canedit="{{ $attendance_edit ? 'true' : 'false' }}" />
+            </div>
+          </div>
+          @if( !empty($images) )
+            <?php
+              $collected_images = [];
+              foreach ($images as $image) {
+                $collected_images[] = $image;
+              }
+            ?>
+            <div class="vue">
+              <EventImages :images="{{ json_encode($collected_images)}}" />
+            </div>
+          @endif
 
-            <h2>@lang('events.event_details')</h2>
-            <div class="card events-card">
+          @if( $event->isInProgress() || $event->hasFinished() )
```

Comment:
Isn't this from the previous work on event stats?  Just wondering why it's popping up in this pull request.

Reply:
You're right, this shouldn't be in this arm of code, as it doesn't apply to upcoming events anyway.  Will remove.